resources/harpy.yaml

```
@@ -4,13 +4,13 @@ channels:
-  - rich-click
-  - snakemake-minimal >7
-  - samtools =1.20
-  - seqtk
-  - tabix
+  - htslib =1.21
+  - pysam =0.23
+  - python >=3.11
+  - rich-click =1.8
+  - snakemake-minimal =9
+  - samtools =1.21
+  - seqtk
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)